app/dom.js

```diff
@@ -183,7 +191,7 @@ const getPaneWidth = () => {
 
 const startDragging = event => {
 	event.preventDefault();
-	const host = getResizeableElement();
+	// Const host = getResizeableElement();
```

Comment:
If it is not a file or large chunks of code. Can we remove this than commenting?

Reply:
Yeah, we totally should!